src/main/kotlin/com/example/mylotto/view/InputView.kt

```diff
@@ -17,4 +17,33 @@ class InputView {
         require(numbers != null)
         return numbers
     }
+
+    fun readBonusNumber(): Int {
+        println("보너스 볼을 입력해 주세요.")
+        val number = readlnOrNull()?.toIntOrNull()
+        require(number != null)
```

Comment:
```suggestion
        require(number != null) { "보너스볼이 입력되지 않았습니다" }
```
요렇게 예외 메세지를 함께 표시해주면 어떨까요?

Reply:
엇.. 넵 감사합니다! 반영하겠습니다~